src/inference/src/system_conf.cpp

```diff
@@ -172,6 +172,24 @@ std::vector<int> get_available_numa_nodes() {
 int get_number_of_logical_cpu_cores(bool) {
     return parallel_get_max_threads();
 }
+
+#    ifdef __APPLE__
```

Comment:
1. Can we have fully separate branches for \_\_APPLE\_\_ and \_\_EMSCRIPTEN\_\_. For now code reuse level is low, but mess of ifdefs doesn't help to understand what's going on.
2. ifdef \_\_APPLE\_\_ -> defined(\_\_APPLE\_\_) (here and in other places)

Reply:
updated